jira/client.py

```diff
@@ -3157,3 +3188,13 @@ def __init__(self, options=None, basic_auth=None, oauth=None, async=None):
             "GreenHopper() class is deprecated, just use JIRA() instead.", DeprecationWarning)
         JIRA.__init__(
             self, options=options, basic_auth=basic_auth, oauth=oauth, async=async)
+
+
+def python_identifier(fieldstring):
+    return re.sub('[^0-9a-zA-Z_]', '', fieldstring.replace(' ', '_')).lower()
+
+def list_lookup(lst, key):
```

Comment:
Good question. Here's why:
customfield_1234 = cf[1234] = "Release Note"

This is a multi-to-one mapping, which a dictionary can't handle properly (no duplicate keys). So a set of lists with equivalence sub-tuples seemed the best way to handle this. Any better solution ideas?